### ISSUE TITLE
A11y_code.visualstudio.com_API Working With Extensions_Continuous Integration_Keyboard: Tooltip is not visible on "Dismiss update button" for keyboard users. 

### ISSUE THREAD

**Issue by RohitJadhav-TCS** [microsoft/vscode-docs#3707]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-manai%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_Code.visualstudio.com_Benchmark;#A11ySev2;#A11yTCS;#A11yMAS;#Benchmark;#ChromiumEdge;#Chrome;#DesktopWeb;#MAS1.1.1;#Win10;#Visual Studio Code;#Keyboard; #CAITestPass_May2020;#FTP;#STP;

**Environment Details**:
Application Name: code.visualstudio.com
#URL:https://code.visualstudio.com
Windows Version: Windows10
Browser version: Microsoft Edge Version 84.0.516.1 (Official build) dev (64-bit)

**Repro Steps**:
1. Launch the application URL :https://code.visualstudio.com
2. Navigate to "API" link.
3. From left navigation, select "Working With Extensions".
4. Navigate to "Continuous Integration" link.
5. Navigate to Search box using tab key.

**Actual**:
Tooltip is not visible on "Dismiss update button" for keyboard users.

**Similar issue observed in through out the application where "Dismiss update button" is present.**

**Expected**:
Tooltip should be visible when keyboard focus goes on that control.

**Attachment link for Reference**:
![3707_A11y_code visualstudio com_API Working With Extensions_Continuous Integration_Keyboard Tooltip is not visible on Dismiss update button for keyboard users](https://user-images.githubusercontent.com/52035395/83004471-58ae1e80-a02d-11ea-9c45-28019fbf6d77.gif)

**Comment by JacksonKearl** [comment]
This behavior is platform-dependent - on Windows/Edge I do get the tooltip on keyboard focus, but this is not a native MacOS behavior. Closing as as-designed/upstream.

![asf](https://user-images.githubusercontent.com/8586769/87476574-733b5780-c5db-11ea-9ab7-7cfaac5c225e.gif)

**Comment by Rama8345** [comment]
Verified the issue at https://code.visualstudio.com/api/working-with-extensions/continuous-integration. The issue is fixed. So closing the bug. Please find below attachment for reference. Thanks
**Github Tags:**#Closed;
![3707_Closed](https://user-images.githubusercontent.com/115216656/219323108-30c26ae0-dcdd-43f1-af99-7971241705ee.gif)
